Makefile

```diff
@@ -69,7 +69,8 @@ DOCKER_PULL := --pull
 
```

Comment:
Please change the commit subject and body to as we try to avoid having commits without any description and reasoning
```
policy: Add template policy

This is an example policy that can be used to develop a new
policy with proper functionality.
```